src/ApparentHorizons/Expansion.hpp

```diff
@@ -0,0 +1,29 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/Tensor/Tensor.hpp"
```

Comment:
I think this included can be move to the cpp? Tensor is forward declared within TypeAliases